.npmignore

```diff
@@ -1,8 +1,14 @@
-.DS_Store
 *.log
-examples
-test
-coverage
-_book
-book.json
-docs
+/.*/
```

Comment:
This is .npmignore not .gitignore. Otherwise I left the same comment as you for elementalUI last week ;)